test/integration/Provider.test.ts

```diff
@@ -27,7 +27,11 @@ describe('Provider tests', async () => {
 
   it('Alice checks fileinfo', async () => {
```

Comment:
this should now check for all possible filetypes, not just URLs. So how about an arweave file test here?

Reply:
Testing all possible file types is out of scope for this PR, but I can add a test for Arweave.

Reply:
Done.